goserver/server/server.go

```diff
@@ -37,9 +46,18 @@ func (s *Server) handleLaunch(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "missing iss in URL query parameters", http.StatusUnauthorized)
 		return
 	}
-	if iss[0] != s.AuthorizedFHIRURL {
+	if iss[0] != s.authorizedFHIRURL {
 		http.Error(w, fmt.Sprintf("unauthorized iss %s", iss[0]), http.StatusUnauthorized)
 		return
 	}
+
+	sess, err := session.New(s.sm, w, r)
+	if err != nil {
+		http.Error(w, "cannot create session", http.StatusBadRequest)
```

Comment:
Is a 400 error appropriate here? Isn't this a server error?

Reply:
Done.

Reply:
Sorry, forgot to change it.